lightning/src/ln/functional_test_utils.rs

```diff
@@ -82,17 +82,12 @@ pub fn confirm_transaction_at<'a, 'b, 'c, 'd>(node: &'a Node<'b, 'c, 'd>, tx: &T
 	connect_block(node, &block, conf_height);
 }
 
-pub fn connect_blocks<'a, 'b, 'c, 'd>(node: &'a Node<'b, 'c, 'd>, depth: u32, height: u32, parent: bool, prev_blockhash: BlockHash) -> BlockHash {
-	// The next commit drops the height, parent, and prev_blockhash parameters. In order to
-	// demonstrate that they are currently unused, we assert that they always match the new default
-	// parameters here.
-	if parent { assert_eq!(prev_blockhash, node.best_block_hash()); }
-	assert_eq!(node.best_block_info().1, height);
-
+pub fn connect_blocks<'a, 'b, 'c, 'd>(node: &'a Node<'b, 'c, 'd>, depth: u32) -> BlockHash {
```

Comment:
Could you also remove `height` from `connect_block` and `disconnect_block`? These are also called in the tests.